client/src/resources/effect/data.rs

```diff
@@ -0,0 +1,29 @@
+//! Contains effects related to the management of data.
+
+/// The type of change that was performed
+#[derive(Debug, Deserialize, Clone)]
+pub enum Kind {
+    /// Data was added to an account.
+    Created(Effect),
+    /// Data was removed from an account.
+    Removed(Effect),
+    /// Data was modified on an account.
+    Updated(Effect),
+}
```

Comment:
So this effect simply states that data was created, destroyed or updated, but no details beyond that.  Interesting 🤔

Reply:
As far as the data that I saw, yes.